src/web/app/pages-instructor/instructor-course-edit-page/copy-instructors-from-other-courses-modal/copy-instructors-from-other-courses-modal-model.ts

```diff
@@ -0,0 +1,28 @@
+import { Instructor } from '../../../../types/api-output';
+import { SortBy, SortOrder } from '../../../../types/sort-properties';
+
+/**
+ * The model for a course tab.
+ */
+export interface CourseTabModel {
+  courseId: string;
+  courseName: string;
+  creationTimestamp: number;
+  isArchived: boolean;
+
+  instructorCandidates: InstructorToCopyCandidateModel[];
+  instructorCandidatesSortBy: SortBy;
+  instructorCandidatesSortOrder: SortOrder;
+
+  hasInstructorsLoaded: boolean;
+  isTabExpanded: boolean;
+  hasLoadingFailed: boolean;
+}
+
+/**
+ * The model for an instructor to copy.
+ */
+export interface InstructorToCopyCandidateModel {
```

Comment:
do we need a new interface? or a Map<Instructor, boolean> would suffice?

Reply:
This entire feature was actually copied from `CopyQuestionsFromOtherSessionsModalComponent`, with some modifications.

Reply:
If you ask my personal opinion, I think the new interface seems neater and more maintainable. :-)